test/server/cards/01_SOR/units/WedgeAntillesStarOfTheRebellion.spec.ts

```diff
@@ -0,0 +1,55 @@
+describe('Wedge Antilles, Star of the Rebellion', function() {
+    integration(function(contextRef) {
+        describe('Wedge Antilles\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['alliance-xwing', 'battlefield-marine', 'clan-saxon-gauntlet'],
+                        groundArena: ['wedge-antilles#star-of-the-rebellion'],
+                        leader: 'hera-syndulla#spectre-two',
+                        resources: 30
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                        spaceArena: ['hwk290-freighter'],
+                        hand: ['atst']
+                    }
+                });
+            });
+
+            it('should give Ambush and +1/+1 to a friendly VEHICLE unit', function () {
```

Comment:
Hey thanks for the comment. You definitely can! The benefit of splitting them up is it given you better insight into what break when it breaks, which is why I opted for a separate test per use case in the suite. For example, if the problem is that new code breaks the matcher and ambush is now given **enemy** units, then the title of the test will make that clear without having to read the specific lines of code.

That being said, it's just a choice that's more in line with what my teams follow at work. If we prefer to club tests I can make that change, but just wanted to given an explanation on why they are separate in this first revision. Just lmk.

Reply:
Yeah I'm normally much more in favor of tests being as separated out as possible, but I've been asking the gang to consolidate tests because the jasmine tests are super heavy and slow to start up for some reason and we have finite github actions time with the free tier. It definitely does make the test failures a little harder to diagnose so it's not optimal for sure.

We do have a plan to eventually refactor to a better test framework, but that just didn't make the cut for the first release unfortunately. You can check out other tests like Agent Kallus to see the conventions we use to try and keep the test cases as clean as possible if they end up being bundled.